src/workerd/server/workerd.capnp

```diff
@@ -84,6 +84,11 @@ struct Config {
   # A list of gates which are enabled.
   # These are used to gate features/changes in workerd and in our internal repo. See the equivalent
   # config definition in our internal repo for more details.
+
+  structuredLogging @5 :Bool = false;
+  # If true, logs will be emitted as JSON for structured logging.
+  # When false, logs use the traditional human-readable format.
+  # This affects the format of logs from KJ_LOG and exception reporting as well as js logs.
```

Comment:
It should probably be called out that this won't work for service worker syntax workers that are using the original module registry implementation.